test/unit/spec/v2/ion/i18nTransformer_spec.js

```diff
@@ -483,7 +483,7 @@ describe('v2/ion/i18nTransformer', function () {
           uiSchema: [
             {
               'name': 'credentials.questionKey',
-              'label': 'Where did you go for your favorite vacation?',
+              'label': 'unit test - vacation location answer',
```

Comment:
do you have test for customized security question and it can not be overwritten?

Reply:
yeah you wrote a test that covers it i think.

Reply:
added a test